terms.html

```diff
@@ -153,8 +153,11 @@
   </dd>
   <dt><dfn data-lt="verify|verified|verifying|verifiable|verifiability">verification</dfn></dt>
   <dd>
-The evaluation of whether a <a>verifiable credential</a> or 
-<a>verifiable presentation</a> complies with this specification.
+The evaluation of whether a <a>verifiable credential</a> or <a>verifiable presentation</a> 
+    is an authentic and timely statement of the issuer (or presenter for a presentation). 
+    This means the credential is conformant to the specification, the proof method is
+    satisfied, AND, if present, the status check succeeds.
+
```

Comment:
I suggest some small tweaks to the whole paragraph...

```pre
The evaluation of whether a <a>verifiable credential</a> (or <a>verifiable presentation</a>) 
is an authentic and timely statement of the issuer (or presenter). This includes checking 
that: the credential (or presentation) conforms to the specification; the proof method is
satisfied; AND, if present, the status check succeeds.
```

The first sentence might be better as --
```pre
The evaluation of whether a <a>verifiable credential</a> or <a>verifiable presentation</a> 
is an authentic and timely statement of the issuer or presenter, respectively. This includes 
checking that: the credential (or presentation) conforms to the specification; the proof 
method is satisfied; AND, if present, the status check succeeds.
```

Reply:
Agreed.